project/project.go

```diff
@@ -18,6 +18,11 @@ import (
 // ComposeVersion is name of docker-compose.yml file syntax supported version
 const ComposeVersion = "1.5.0"
 
+var (
+	// ErrNoConfig is returned when a service config cannot be found by name
+	ErrNoConfig = errors.New("Failed to find service")
```

Comment:
I'm not found of exported errors as they couple the caller (or the code handling the error) to the library :stuck_out_tongue_closed_eyes: (cf http://dave.cheney.net/2016/04/27/dont-just-check-errors-handle-them-gracefully)